appveyor.yml

```diff
@@ -293,9 +293,13 @@ build_script:
       )))))
 
 test_script:
+    - cmd: set TFLAGS=%DISABLED_TESTS%
     - if %TESTING%==ON (
-        echo APPVEYOR_API_URL=%APPVEYOR_API_URL% &&
-        bash.exe -e -l -c "cd /c/projects/curl/tests && ./runtests.pl -a -b$(($(echo '%APPVEYOR_API_URL%' | cut -d'/' -f3 | cut -d':' -f2)+1)) -p !flaky %DISABLED_TESTS%" )
+        if %BUILD_SYSTEM%==CMake (
+          cmake --build . --target test-nonflaky
```

Comment:
It seems like the tests are build during this step, right? Could this maybe be moved to the build step and just run them here?

Reply:
@mback2k Yes. It was a significant change #5368 and tests were removed from as dependency from target all.
We can try to add an additional step for the build. Something like 
```
  if %TESTING%==ON (
       cmake --build . --target testdeps
  )
```
It can do the trick.

Reply:
done